src/pages/home-page.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {View, Text, StyleSheet} from 'react-native';
+
+export const HomePage: React.FC<{navigation: any}> = () => {
```

Comment:
```suggestion
export const HomePage: React.FC = () => {
```

Nit: dado que você não tá utilizando esse parâmetro, talvez não seja necessário declarar, ainda mais se for como `any`, que algo que queremos evitar ao usar Typescript.